x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.test.ts

```diff
@@ -514,6 +601,110 @@ describe('Log threshold executor', () => {
           },
         ]);
       });
+
+      it('respects and reports reaching a low limit when alerting', () => {
```

Comment:
I'm liking these new tests. Makes easier to see what's going in and compare it to the asserted results.